### ISSUE TITLE
Mapset deletion bug

### ISSUE THREAD

**Issue by anikaweinmann** [mundialis/actinia_core#339]
The demouser can create new mapsets.
Which can be listed with:
```
curl -u 'demouser:gu3st!pa55w0rd' -X GET "https://actinia.mundialis.de/api/v3/locations/nc_spm_08/mapsets" | jq
```
The try to delete a generated mapset e.g. `test_AW_JHA_VK` fails:
```
curl -u 'demouser:gu3st!pa55w0rd' -X DELETE  "https://actinia.mundialis.de/api/v3/locations/nc_spm_08/mapsets/test_AW_JHA_VK" | jq
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   236  100   236    0     0    631      0 --:--:-- --:--:-- --:--:--   631
{
  "message": "The server could not verify that you are authorized to access the URL requested. You either supplied the wrong credentials (e.g. a bad password), or your browser doesn't understand how to supply the credentials required."
}
```
The demouser should be allowed to delete his self-created mapset again, otherwise they accumulate.
Wanted features:

- mapset management:
  -  user is able to create own mapsets
  -  user is able to delete own mapsets (mind group rights)
- persisten processing:
  -  user is able to delete own mapsets (mind group rights)
  - additionally create cleanup mechanism (automatically delete XYZ from demouser when?, e.g. every 24h)

**Comment by juleshaas** [comment]
Has been partly solved.
Mapset management: solved in #365
Persistent processing: still TBD

**Comment by juleshaas** [comment]
Deletion of mapsets created via persistent processing has been tested successfully.
Weekly cronjob for [actinia.mundialis.de](actinia.mundialis.de) and [actinia-dev.mundialis.de](actinia-dev.mundialis.de) will be implemented to delete locations and mapsets older than 30 days from user DB.
